app/controllers/provider_interface/content_controller.rb

```diff
@@ -12,8 +12,18 @@ def accessibility
       render_content_page :accessibility
     end
 
-    def privacy_policy
-      render_content_page :privacy_policy
+    def privacy; end
+
+    def service_privacy_notice
+      render_content_page :service_privacy_notice,
+                          breadcrumb_title: 'privacy_notices',
+                          breadcrumb_path: provider_interface_privacy_path
+    end
+
+    def online_chat_privacy_notice
+      render_content_page :online_chat_privacy_notice,
+                          breadcrumb_title: 'privacy_notices',
```

Comment:
```suggestion
                          breadcrumb_title: 'online_chat_privacy_notice',
```

Reply:
This is the title for the previous page in the breadcrumb - the current page is drawn from the current page title, but in this case the previous page title should be privacy notices. 
![image](https://user-images.githubusercontent.com/25597009/144033100-22e076aa-4cfc-4948-a27f-e9ab0498ed69.png)